src/components/login.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react'; // Importa React e o hook useState
+
+// Define o componente Login que recebe uma função onLogin como prop
+const Login = ({ onLogin }) => {
+    // Define estados para email e senha, ambos inicialmente vazios
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+
+    // Função que é chamada quando o formulário é enviado
+    const handleSubmit = (e) => {
+        e.preventDefault(); // Previne o comportamento padrão do formulário (recarregar a página)
+        onLogin({ email, password }); // Chama a função onLogin com os valores de email e senha
+    };
+
+    return (
+        // Formulário que chama handleSubmit quando enviado
+        <form onSubmit={handleSubmit}>
+            <h2>Login</h2>
+            <label>
```

Comment:
Não falto indicar a que input essa label pertence? No html a gente usa o for para indicar a quem a label pertence, no react é o htmlFor

Reply:
vejam esse site: https://developer.mozilla.org/pt-BR/docs/Learn/Forms/How_to_structure_a_web_form